index.js

```diff
@@ -31,7 +35,17 @@ getFullNames(usersArray);
 // ***************************************************************************
 
 const getUsersCreditDetails = arr => {
-  // Your code goes here ...
+  const userCreditDetails = [];
+  for (let user of arr){
+    let {firstName = user.firstName, lastName = user.lastName, balance = user.balance} = arr;
```

Comment:
this shouldn't work, to be honest.

You are trying to object destructure an array in this line.

The fact you already have are doing the for loop you can directly
```js
let { firstName, lastName, balance } = user;
```
you are trying to destructure an object from an array and it might throw an error, but because you give the fallback / default values, it might be passing.
Does this make sense?